File: Code/C_Code/I2C_CR.c
#include <stdio.h>
#include <stdlib.h>
#include <stdarg.h>
#include <unistd.h>
#include <pigpio.h>

/*
 * Code developed using pigpio library. 
 * For more information visit: http://abyz.me.uk/rpi/pigpio/cif.html
 * to compile using this library: 
 * gcc -Wall -pthread -o program.bin program.c -lpigpio -lrt
*/


int main(int argc, char *argv[])
{
   int handle;
   if (gpioInitialise()<0)
   {
      printf("Initializing GPIO Library Failed\n \r");
      return 1;
   } 
   handle = i2cOpen(1, 0x3F, 0);
   printf("Sending Data \n");
   i2cWriteByte(handle, 0xAA);
   printf("Program End\n");
   gpioTerminate();
   
}

File: Code/Python_Code/I2C_CR.py
#Using SMBUS2 Library. For more information:
#https://smbus2.readthedocs.io/en/latest/
import smbus2

def main():
    print("Initializing I2C Bus")
    bus = smbus2.SMBus(1)
    print("Writing Byte")
    bus.write_byte(0x3F,0xAA)

if __name__ == "__main__":
    main()

